components/email/src/Email.svelte

```diff
@@ -75,6 +76,10 @@
     // Fetch Account
     if (id && !you.id && !emailManuallyPassed) {
       you = await fetchAccount({ component_id: query.component_id });
+      userEmail =
+        !you.email_address || userEmail !== you.email_address
+          ? you.email_address
+          : userEmail;
```

Comment:
If `you.email_address` is falsey OR `userEmail` doesn't match `you.email_address` then set it to `you.email_address` or keep it the same as `userEmail`